dashboard/test/controllers/aichat_controller_test.rb

```diff
@@ -9,14 +9,18 @@ class AichatControllerTest < ActionController::TestCase
     @genai_pilot_teacher = create :teacher, pilot_experiment: @genai_pilot.name
     pilot_section = create(:section, user: @genai_pilot_teacher)
     @genai_pilot_student = create(:follower, section: pilot_section).student_user
+    @genai_pilot_teacher2 = create :teacher, pilot_experiment: @genai_pilot.name
+    @level = FactoryBot.create(:level, name: 'level1')
```

Comment:
Is this the same as saying `@level = create :level, name: 'level1'`?

Reply:
Oh yeah! I'll simplify.